components/filter/filter.js

```diff
@@ -43,10 +47,28 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			div[slot="header"] {
 				padding: 0.9rem 0.3rem;
 			}
+
 			.d2l-filter-dimension-header {
+				padding-bottom: 0.9rem;
+			}
+
+			.d2l-filter-dimension-header,
+			.d2l-filter-dimension-header-actions {
 				align-items: center;
 				display: flex;
 			}
+
+			d2l-input-search {
+				flex-grow: 1;
+				padding-left: 0.3rem;
+				padding-right: 0.6rem;
```

Comment:
Do margins still get you the same effect? I feel weird about setting padding on a thing where that doesn't impact the space inside its borders.

Reply:
Yup I can switch to margins here